RiotSwiftUI/Modules/LocationSharing/LiveLocationSharingViewer/LiveLocationSharingViewerViewModel.swift

```diff
@@ -229,4 +232,14 @@ class LiveLocationSharingViewerViewModel: LiveLocationSharingViewerViewModelType
             }
         }
     }
+    
+    private let locationManager = CLLocationManager()
```

Comment:
I think that this view model shouldn't know about CoreLocation.
As you can see there is already a service dealing with location (`var liveLocationSharingViewerService: LiveLocationSharingViewerServiceProtocol`).
Is it worth moving this logic there?
This improves testability as well. ;)

Reply:
Agree and done, should I create also a service for StaticLocation since it doesn't exist or it should be fine in the viewModel?

Reply:
It would be a nice to have.
It's up to you!